pty/pty_other.go

```diff
@@ -95,13 +100,21 @@ func (p *otherPty) Input() ReadWriter {
 	}
 }
 
+func (p *otherPty) InputWriter() io.Writer {
+	return p.pty
+}
+
 func (p *otherPty) Output() ReadWriter {
 	return ReadWriter{
-		Reader: p.pty,
+		Reader: &ptmReader{p.pty},
 		Writer: p.tty,
```

Comment:
We close & nil the `tty` on a call to `pty.Start()`, which returns a `PTYCmd`, which doesn't have the `Output()` method.  Preventing a panic is the primary reason I split it into two different interfaces.

Note that if you `Close()` the `PTY` then you have a closed file, not a `nil` and it will return an error rather than panicking.